language/stdlib/modules/LibraConfig.move

```diff
@@ -165,6 +165,18 @@ module LibraConfig {
 
        emit_reconfiguration_event();
     }
+    spec fun reconfigure_ {
+        /// Consider only states for verification where this function does not abort
```

Comment:
Maybe there shouldn't be asserts for internal program errors.  I don't understand the code for reconfigure_, so that didn't clarify things for me.

Reply:
I want to open a discussion about this after this PR. We should really partition error codes into internal errors (which don't need an aborts_if spec) and others (which are caused by user input). That is also common in languages like Java and C#, where there are checked and unchecked exceptions.